em_examples/DCWidgetPlate_2D.py

```diff
@@ -298,67 +312,135 @@ def sumPlateCharges(xc,zc,dx,dz,rotAng,qSecondary):
     return qPosSum, qNegSum, qPosAvgLoc, qNegAvgLoc
 
 
-# Inline functions for computing apparent resistivity
-eps = 1e-9 #to stabilize division
-G = lambda A, B, M, N: 1. / ( 1./(np.abs(A-M)+eps) - 1./(np.abs(M-B)+eps) - 1./(np.abs(N-A)+eps) + 1./(np.abs(N-B)+eps) )
-rho_a = lambda VM,VN, A,B,M,N: (VM-VN)*2.*np.pi*G(A,B,M,N)
+def getSensitivity(survey,A,B,M,N,model):
```

Comment:
@thast: do you have a linter on for python? https://github.com/SublimeLinter/SublimeLinter-pep8

In the future, if you would use that, that would be helpful. It does not need to be implemented for this pr though